src/Infrastructure/Repositories/LocalFilesRepository.php

```diff
@@ -56,6 +56,12 @@ public function within(string $directory, array $exclude): FilesRepository
         $this->finder->in([$directory])->exclude($exclude);
```

Comment:
I'm not sure, but I think if we replace `exclude` by `notPath` here, the foreach will be not needed and `directory` and `sub/file.php` will be excluded.

Reply:
I'll try this ASAP, thanks :)

Reply:
Indeed, I missed this function. Tests are still passing with it, so it's awesome :)